packages/contracts/deploy/update/to_v1.3.0/30_AddresslistVoting_Plugin.ts

```diff
@@ -0,0 +1,90 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {PluginRepo__factory} from '../../../typechain';
+import {getContractAddress, uploadToIPFS} from '../../helpers';
+
+import addresslistVotingSetupArtifact from '../../../artifacts/src/plugins/governance/majority-voting/addresslist/AddresslistVotingSetup.sol/AddresslistVotingSetup.json';
+import addresslistVotingReleaseMetadata from '../../../src/plugins/governance/majority-voting/addresslist/release-metadata.json';
+import addresslistVotingBuildMetadata from '../../../src/plugins/governance/majority-voting/addresslist/build-metadata.json';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  console.log('\nUpdate AddresslistVoting Plugin');
+  const {deployments, ethers, network} = hre;
+  const {deploy} = deployments;
+  const [deployer] = await ethers.getSigners();
+
+  const deployResult = await deploy('AddresslistVotingSetup', {
+    contract: addresslistVotingSetupArtifact,
+    from: deployer.address,
+    args: [],
+    log: true,
+  });
+
+  const addresslistVotingReleaseCIDPath = await uploadToIPFS(
+    JSON.stringify(addresslistVotingReleaseMetadata),
+    network.name
+  );
+  const addresslistVotingBuildCIDPath = await uploadToIPFS(
+    JSON.stringify(addresslistVotingBuildMetadata),
+    network.name
+  );
+
+  const addresslistVotingRepoAddress = await getContractAddress(
+    'address-list-voting-repo',
+    hre
+  );
+  const addresslistVotingRepo = PluginRepo__factory.connect(
+    addresslistVotingRepoAddress,
+    ethers.provider
+  );
+  if (
+    await addresslistVotingRepo.callStatic.isGranted(
+      addresslistVotingRepoAddress,
+      deployer.address,
+      await addresslistVotingRepo.MAINTAINER_PERMISSION_ID(),
+      '0x00'
+    )
+  ) {
+    console.log(
+      `Deployer has permission to install new AddresslistVoting version`
+    );
+    const tx = await addresslistVotingRepo
+      .connect(deployer)
+      .createVersion(
+        1,
```

Comment:
Same as before

Reply:
Added in bd20b893734d9afa715bf68128e4514ffeed67ea